### PR TITLE
Link in Models to DB class returning 404 & Model documentation link

### PR DESCRIPTION
**Pull request by zoe-edwards**
I know I edited one of them a month or two ago, but the other one needed fixing too. Didn’t spot it the first time.

Added an extra link for those reading through the model docs confused to where to turn to create their first model.

### CODE REVIEW

**Comment by zoe-edwards** [comment]
Sorry total idiot, added a space in for funsies.

**Comment by zoe-edwards** [comment]
Hey look at that, Git’s quite clever!

File: general/models.html [hunk 1]
```diff
@@ -34,6 +34,10 @@
 			call upon the Model to execute the queries. This way if your database changes you won't need to change all your Controllers but just the Model that acts upon it.
 		</p>
 
+		<p>
+			See <a href="/packages/orm/creating_models.html">more about creating models</a> in the Orm Package documentation.
```
Comment (frankdejonge) [inline general/models.html]: This link should be relative from general/mode.html so prefixed with ../

Comment (zoe-edwards) [inline general/models.html]: That’s twice I’ve missed that!

**Comment by zoe-edwards** [comment]
There we go.

**Comment by frankdejonge** [comment]
And one more thing... you're requesting a pull on the master branch, we only accept on the develop branch. :) So if you could add it to your develop and request the pull there?

**Comment by zoe-edwards** [comment]
One day I will actually realise what I’m doing.

**Pull request closed without merging**